Formula/ktlint.rb

```diff
@@ -1,23 +1,20 @@
 class Ktlint < Formula
   desc "Anti-bikeshedding Kotlin linter with built-in formatter"
   homepage "https://ktlint.github.io/"
-  url "https://github.com/pinterest/ktlint/releases/download/0.47.0/ktlint"
-  sha256 "36e3641018c559492214ab9dd6c30575890edb3d84034dd73daca50b46316384"
+  url "https://github.com/pinterest/ktlint/releases/download/0.47.1/ktlint"
+  sha256 "a333ad0172369a5cd973aea83e02e8b698c06a2daac6f32925da03049aa3dce7"
   license "MIT"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "017fc6bb39413ac029b64af8e0ba20c89f7e5a8f1779653a534f93d7ec0af36f"
   end
 
-  depends_on "openjdk@11"
+  depends_on "openjdk"
 
   def install
     libexec.install "ktlint"
     (libexec/"ktlint").chmod 0755
-    (bin/"ktlint").write_env_script libexec/"ktlint",
-                                    Language::Java.java_home_env("11").merge(
-                                      PATH: "#{Formula["openjdk@11"].opt_bin}:${PATH}",
-                                    )
+    (bin/"ktlint").write_env_script libexec/"ktlint", Language::Java.java_home_env
```

Comment:
Might still need `PATH`, since `openjdk` is also keg-only.

Reply:
I see.

> Test seems happy though.

probably because `shadowed_by_macos`?

Reply:
never mind, after a bit reading, I dont think PATH change is needed.